src/index.css

```diff
@@ -4,9 +4,16 @@ body {
   font-family: sans-serif;
 }
 
-.screen-content {
-  height: calc(100vh - 120px);
+.app > .screen-content {
+  height: calc(100vh - 72px);
```

Comment:
It's always better to leave comments for magic numbers like this, cause
"// When I wrote this, only God and I understood what I was doing.
// Now only God knows".